src/main/java/mate.academy/Main.java

```diff
@@ -4,13 +4,16 @@
 import mate.academy.lib.Injector;
 import mate.academy.model.Product;
 import mate.academy.service.ProductService;
+import mate.academy.service.impl.ProductServiceImpl;
 
 public class Main {
 
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        ProductService productService = (ProductServiceImpl) injector.getInstance(
+                ProductService.class
+        );
```

Comment:
Casting the `ProductService` instance to `ProductServiceImpl` may lead to a `ClassCastException` if the injector does not return an instance of `ProductServiceImpl`. Ensure that the injector is configured to return the correct implementation or handle the casting more safely.